packages/web-components/src/switch/switch.styles.ts

```diff
@@ -0,0 +1,173 @@
+import { css } from '@microsoft/fast-element';
+import {
+  colorCompoundBrandBackground,
+  colorCompoundBrandBackgroundHover,
+  colorCompoundBrandBackgroundPressed,
+  colorNeutralBackgroundDisabled,
+  colorNeutralForeground1,
+  colorNeutralForeground3,
+  colorNeutralForeground3Hover,
+  colorNeutralForeground3Pressed,
+  colorNeutralForegroundDisabled,
+  colorNeutralForegroundInverted,
+  colorNeutralForegroundInvertedHover,
+  colorNeutralForegroundInvertedPressed,
+  colorNeutralStrokeAccessible,
+  colorNeutralStrokeAccessibleHover,
+  colorNeutralStrokeAccessiblePressed,
+  colorNeutralStrokeDisabled,
+  colorTransparentBackground,
+  curveEasyEase,
+  durationNormal,
+  fontFamilyBase,
+  fontSizeBase300,
+  fontWeightSemibold,
+  lineHeightBase300,
+  spacingHorizontalS,
+  spacingVerticalS,
+  spacingVerticalXS,
+} from '../theme/design-tokens.js';
+
+export const styles = css`
+  :host([hidden]) {
+    display: none;
+  }
+  :host {
```

Comment:
The styles slightly differ compared to FUIR9 (label font, dot vertical alignment, spacing between label and the oval):

FUIR9
![image](https://user-images.githubusercontent.com/9615899/215533918-3702176a-8e80-402d-a08e-36e7e0d23210.png)

FUIWC3
![image](https://user-images.githubusercontent.com/9615899/215533992-f5302272-205e-48e9-813c-37c7c110740b.png)

Reply:
@miroslavstastny which browser are you viewing these in? I'm not seeing the alignment issues in chrome or edge.

FUIWC3
![image](https://user-images.githubusercontent.com/11062709/215560862-de71486a-cd09-48f8-95ed-2c7081c6d839.png)

Reply:
Any browser on Mac OS Ventura 13.2.
Firefox 109.0:
<img width="194" alt="image" src="https://user-images.githubusercontent.com/9615899/215760192-cfdeba1d-b3c0-4b5d-b167-e937cbb10394.png">

Chrome 109.0.5414.119:
<img width="173" alt="image" src="https://user-images.githubusercontent.com/9615899/215760302-2dd16635-0c64-4041-8a8f-6bce24102fd1.png">

Edge 109.0.1518.70:
<img width="178" alt="image" src="https://user-images.githubusercontent.com/9615899/215760557-41c9d803-5059-44c3-84d9-164048bc7cf2.png">

Safari 16.3 (18614.4.6.1.5):
<img width="170" alt="image" src="https://user-images.githubusercontent.com/9615899/215760695-1c1a86b2-b47e-49ce-9db3-55d7ccfb65c9.png">

Reply:
The same in Edge 109.0.1518.70 on Windows 11:
<img width="177" alt="image" src="https://user-images.githubusercontent.com/9615899/215763674-0ab25eeb-177c-421d-a044-fe2b0561bfb1.png">

Reply:
@miroslavstastny Okay, made some changes..

**Chrome on 13.2**

![MicrosoftTeams-image (3)](https://user-images.githubusercontent.com/11062709/215904242-7b3442da-7da7-46d5-9aec-6fb783ea21a3.png)

**Edge on 13.2**

![MicrosoftTeams-image (4)](https://user-images.githubusercontent.com/11062709/215904328-0acab6f4-14a1-4a09-be39-a150f1fb5b1b.png)